### PR TITLE
Fix execution instruction in readme

### PR DESCRIPTION
**Pull request by maximkoretskiy**
In fact, this file is not binary, it's a script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -449,10 +449,10 @@ If you installed Cucumber.js globally, you may run it with:
 $ cucumber.js
 ```
 
-If you installed Cucumber locally, you may need to specify the path to the binary:
+If you installed Cucumber locally, you may need to specify the path to the executable:
 
 ``` shell
-$ ./node_modules/.bin/cucumber.js
+$ ./node_modules/bin/cucumber.js
```
Comment (charlierudolph) [inline README.md]: the `.bin` is still needed as that is where npm puts executables.

**Comment by charlierudolph** [comment]
After the one comment is addressed, this looks good. Thanks!

**Comment by maximkoretskiy** [comment]
It looks like npm has different behavior in different operation systems. May be it depends on npm and node versions too. I need to find out the most efficient recipe.

**Comment by maximkoretskiy** [comment]
Sorry, my bad. You a right.

**Comment by maximkoretskiy** [comment]
@charlierudolph done

**Comment by charlierudolph** [comment]
Thanks!

**Pull request closed without merging**